PopulateTools/gobierto
# Custom I18n backend for Gobierto

Closes #1222

### What does this PR do?

This PR introduces a new I18n backend persisted in the databased, and based on `I18n::Backend::ActiveRecord` but with the feature of support multitenant translations.

### How should this be manually tested?

1. Create a custom translation in a site.
2. Check the translation is visible in that site
3. Check the translation is not visible in other sites.
